charts/spire/charts/tornjak-frontend/Chart.yaml

```
@@ -4,6 +4,10 @@ description: A Helm chart to deploy Tornjak frontend
 type: application
 version: 0.1.0
 appVersion: "v1.0.2"
+home: https://github.com/spiffe/helm-charts/tree/main/charts/spire
+sources:
+  - https://github.com/spiffe/tornjak
+icon: https://github.com/spiffe/tornjak/blob/main/logos/logo%2Btornjak.2132x1291.png
```

I think the other url is to the web page with the image embedded. 
```suggestion
icon: https://raw.githubusercontent.com/spiffe/tornjak/main/logos/logo%2Btornjak.2132x1291.png
```